src/core/core/perf_counters.cpp

```diff
@@ -139,6 +139,13 @@ perf_counters::perf_counters(void)
         "counter.samplei counter-index",
         &perf_counters::get_counter_sample_i
         );
+
+    ::dsn::register_command("counter.getindex",
+        "counter.getindex - get index of a specific counter",
+        "counter.getindex app-name*section-name*counter-name",
```

Comment:
the comment and the implementation are not consistent. Seems your command can support fetch indices for a set of counters.